doc/nrf/getting_started/modifying.rst

```diff
@@ -292,7 +296,9 @@ These options are specified when CMake is run, thus not during the actual build,
 
    .. group-tab:: Command line
 
-      |west_build_option|
+      If you work on the command line, pass the additional options to the ``west build`` command.
+      The options must be added after a ``--`` at the end of the command.
+      See :ref:`zephyr:west-building-cmake-args` for more information.
```

Comment:
@nordicjm , @mbolivar-nordic , re previous comment: this link still points to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/develop/west/build-flash-debug.html#one-time-cmake-arguments

Reply:
Deferring to Marti, I do not use west.